pkg/proto/readme.md

```diff
@@ -0,0 +1,8 @@
+Notes for protobuf
+
+1. Run this command to compile .proto to .pb.go file.
+```/AgentBaker/hack/tools/bin/buf generate -o .```
+
+1. If you haven't installed buf yet, run the following command, assumed /usr/local/bin is where your Go binary locates.
+
+```GO111MODULE=on GOBIN=/usr/local/bin go install github.com/bufbuild/buf/cmd/buf@v1.27.2```
```

Comment:
Devin, we already have a make target to create the buf binary, https://github.com/Azure/AgentBaker/blob/c6fb221ae595a481af6380d777969916e5367634/hack/tools/Makefile#L25

Reply:
updated to use that binary.